### PR TITLE
Add OpenVino face detection sample

### PR DESCRIPTION
**Pull request by vinicius121**
This example is to detect faces using Intel OpenVino Inference Engine.

### CODE REVIEW

File: SamplesCS/Samples/OpenVinoFaceDetection.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        {
+	    const string modelFace = "face-detection-adas-0001.bin"; 
+	    const string modelFaceTxt = "face-detection-adas-0001.xml";
+            const string sampleImage = "sample.jpg";
+            const string outputLoc = "sample_output.jpg";
+
+            using var frame = Cv2.ImRead(sampleImage);
+            int frameHeight = frame.Rows;
+            int frameWidth = frame.Cols;
+	    using var netFace = CvDnn.ReadNet(modelFace, modelFaceTxt);
```
Comment (shimat) [inline SamplesCS/Samples/OpenVinoFaceDetection.cs]: please replace tab with four spaces

**Review by shimat — COMMENTED** [review]
Thank you for your contribution!

File: SamplesCS/Samples/OpenVinoFaceDetection.cs (hunk 1 above)
Comment (vinicius121) [reply SamplesCS/Samples/OpenVinoFaceDetection.cs]: Hi, I think i updated it.

**Comment by shimat** [comment]
Thanks!

**Pull request merged**